warehouse/client/controlplane/client.go

```diff
@@ -0,0 +1,99 @@
+package controlplane
```

Comment:
Would it make sense to move the client [here](https://github.com/rudderlabs/rudder-server/tree/master/services/controlplane)